test-support/ember-sinon-qunit/test.js

```diff
@@ -14,18 +14,24 @@ sinon.assert.pass = function (assertion) {
 sinon.config = {
   injectIntoThis: false,
   injectInto: null,
-  properties: ['spy', 'stub', 'mock', 'sandbox'],
+  properties: ['spy', 'stub', 'mock', 'sandbox', 'clock'],
   useFakeTimers: false,
   useFakeServer: false
 };
 
-export default function test(testName, callback) {
+export default function test(testName, options, callback) {
+  if (arguments.length < 3) {
+    callback = options;
+    options = {};
+  }
+
   function sinonWrapper() {
     let context = this;
     if (Ember.isBlank(context)) {
       context = {};
     }
     sinon.config.injectInto = context;
+    sinon.config.useFakeTimers = !!options.useFakeTimers;
 
     return sinon.test(callback).apply(context, arguments);
```

Comment:
It is the arguments of the `sinonWrapper` function. I wrote a parallel change in `ember-qunit` to leverage this better: https://github.com/rwjblue/ember-qunit/commit/5eb95b5fd34979a8d2e3d5dfe3a912d41d81fdc4